json/circe/src/test/scala/sttp/client4/circe/BackendStubCirceTests.scala

```diff
@@ -6,14 +6,27 @@ import sttp.client4.testing.SyncBackendStub
 import io.circe.generic.auto._
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import io.circe.syntax.EncoderOps
+import io.circe.JsonObject
+import sttp.model.Uri
 
 class BackendStubCirceTests extends AnyFlatSpec with Matchers with ScalaFutures {
+
   it should "deserialize to json using a string stub" in {
     val backend = SyncBackendStub.whenAnyRequest.thenRespond("""{"name": "John"}""")
     val r = basicRequest.get(uri"http://example.org").response(asJson[Person]).send(backend)
     r.is200 should be(true)
     r.body should be(Right(Person("John")))
   }
 
+  it should "serialize from JsonObject using implicit circeBodySerializer" in {
+    val jObject: JsonObject = JsonObject(("location", "hometown".asJson), ("bio", "Scala programmer".asJson))
```

Comment:
one more thing - these are not tests for the stub, they are tests for the integration. So I think these shuold go to `CirceTests` simply